docs/create-packages/Prerelease-Packages.md

```diff
@@ -76,10 +76,10 @@ Whatever suffixes you use, however, NuGet will give them precedence in reverse a
     1.0.1-zzz
     1.0.1-rc
     1.0.1-open
-    1.0.1-beta12
-    1.0.1-beta05
+    1.0.1-beta.12
+    1.0.1-beta.05
```

Comment:
The idea with semver 2 versions is that you don't need the leading 0. 

So you would have. 

`1.0.1-beta.12` and `1.0.1-beta.5`

Reply:
OK, thanks, updating now.